modules/apps/layout/layout-reports-web/jest-setup.config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+window.Liferay.Util.sub = function (string = '', data) {
+	const REGEX_SUB = /(?<=-|^)x(?=-|\s)/g;
+
+	if (
+		arguments.length > 2 ||
+		(typeof data !== 'object' && typeof data !== 'function')
+	) {
+		data = Array.prototype.slice.call(arguments, 1);
+	}
+
+	const dataCopy = [...data];
+	const max = REGEX_SUB.exec(string).length;
```

Comment:
Mini-nit: `max` → `maximum` (per [guidelines](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/naming.md#avoid-abbreviations), abbreviations and all that)

Non-blocking feedback though, especially because this file might go away based on https://github.com/liferay/liferay-frontend-projects/issues/519

Reply:
Thanks for the nit. Hopefully, this file will be removed soon and provide it by default in our tests 🍀